lib/util/usedPropTypes.js

```diff
@@ -285,13 +285,14 @@ module.exports = function usedPropTypesInstructions(context, components, utils)
    * @returns {Boolean} True if we are using a prop, false if not.
    */
   function isPropTypesUsage(node) {
+    const isThisPropsUsage = node.object.type === 'ThisExpression' && node.property.name === 'props';
+    const isPropsUsage = isThisPropsUsage || node.object.name === 'nextProps' || node.object.name === 'prevProps';
```

Comment:
Thanks for the fix!
Later on it'd be great not to depend on the names (similar to #1829), but for now not crashing is the priority 👍

Reply:
Agreed!